README.md

```diff
@@ -1 +1,169 @@
-# tv-webapp
\ No newline at end of file
+# tv-webapp
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# Tv webapp <a name="about-project"></a>
+
+**tv-webapp** is a web application that allows users to like and comment on tv series.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+This project was built using these technologies.
+
+- JavaScript
+- Webpack
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Lets users like series**
+- **Lets users comment on series**
+- **Lets users see the number of likes and comments on each series**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Here](https://LeslieAine.github.io/tv-webapp/)
+
```

Comment:
<img width="860" alt="1" src="https://user-images.githubusercontent.com/77843050/219641452-16bfc326-b25c-4d26-bfa3-9cf5330ca729.png">

- [ ] Hey @LeslieAine, I understand the time and energy you investigate in order to deploy the app but this demo link is not working yet, please take your time to fix this as it is very important for you.